passagemath-cmr wheel [macos]: Library not loaded: @rpath/libgtest_main.1.15.2.dylib

```
    [cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5]   [spkg-install] cd /Users/runner/sage-local/var/tmp/sage/build/cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5/src/build/test && /opt/homebrew/bin/cmake -D TEST_TARGET=cmr_gtest -D TEST_EXECUTABLE=/Users/runner/sage-local/var/tmp/sage/build/cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5/src/build/test/cmr_gtest -D TEST_EXECUTOR= -D TEST_WORKING_DIR=/Users/runner/sage-local/var/tmp/sage/build/cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5/src/build/test -D TEST_EXTRA_ARGS= -D TEST_PROPERTIES= -D TEST_PREFIX= -D TEST_SUFFIX= -D TEST_FILTER= -D NO_PRETTY_TYPES=FALSE -D NO_PRETTY_VALUES=FALSE -D TEST_LIST=cmr_gtest_TESTS -D CTEST_FILE=/Users/runner/sage-local/var/tmp/sage/build/cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5/src/build/test/cmr_gtest[1]_tests.cmake -D TEST_DISCOVERY_TIMEOUT=5 -D TEST_DISCOVERY_EXTRA_ARGS= -D TEST_XML_OUTPUT_DIR= -P /opt/homebrew/share/cmake/Modules/GoogleTestAddTests.cmake
    [cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5]   [spkg-install] dyld[6106]: Library not loaded: @rpath/libgtest_main.1.15.2.dylib
    [cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5]   [spkg-install]   Referenced from: <8B59FD9C-E3CD-316B-8224-338B4F3B990B> /Users/runner/sage-local/var/tmp/sage/build/cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5/src/build/test/cmr_gtest
    [cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5]   [spkg-install]   Reason: tried: '/Users/runner/sage-local/var/tmp/sage/build/cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5/src/build/libgtest_main.1.15.2.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/runner/sage-local/var/tmp/sage/build/cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5/src/build/libgtest_main.1.15.2.dylib' (no such file), '/Users/runner/sage-local/var/tmp/sage/build/cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5/src/build/libgtest_main.1.15.2.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/runner/sage-local/var/tmp/sage/build/cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5/src/build/libgtest_main.1.15.2.dylib' (no such file)
    [cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5]   [spkg-install] CMake Error at /opt/homebrew/share/cmake/Modules/GoogleTestAddTests.cmake:132 (message):
    [cmr-ecd47e4dde3167d7b65017cdbdf1efa33d3dc1f5]   [spkg-install]   Error running test executable.
 ```
https://github.com/passagemath/passagemath/actions/runs/14581182236/job/40911759920#step:12:12581

Same on macos x86_64:
https://github.com/passagemath/passagemath/actions/runs/14581182236/job/40911759923#step:12:12761

--- a/build/pkgs/cmr/spkg-install.in
+++ b/build/pkgs/cmr/spkg-install.in
@@ -4,6 +4,6 @@ cd src
 export CXX="$(echo "$CXX" | sed 's/-std=[a-z0-9+]*//g') -std=gnu++14"
 mkdir build
 cd build
-sdh_cmake -DGENERATORS=on -DCMAKE_SYSTEM_PREFIX_PATH="$SAGE_LOCAL" -DHAVE_FLAG_SEARCH_PATHS_FIRST=0 -DSHARED=on ..
+sdh_cmake -DGENERATORS=on -DCMAKE_SYSTEM_PREFIX_PATH="$SAGE_LOCAL" -DCMAKE_BUILD_RPATH="${SAGE_LOCAL}/lib" -DCMAKE_INSTALL_RPATH="${SAGE_LOCAL}/lib" -DHAVE_FLAG_SEARCH_PATHS_FIRST=0 -DSHARED=on ..
 sdh_make VERBOSE=1
 sdh_make_install

